modules/tinymce/src/plugins/table/main/ts/api/Api.ts

```diff
@@ -5,35 +5,32 @@
  * For commercial licenses see https://www.tiny.cloud/
  */
 
-import Editor from 'tinymce/core/api/Editor';
-import * as InsertTable from '../actions/InsertTable';
-import { Arr, Option, Cell } from '@ephox/katamari';
-import { Element } from '@ephox/sugar';
 import { HTMLElement } from '@ephox/dom-globals';
+import { Arr, Cell, Option } from '@ephox/katamari';
+import { Element } from '@ephox/sugar';
+import Editor from 'tinymce/core/api/Editor';
 import { ResizeHandler } from '../actions/ResizeHandler';
 import { SelectionTargets } from '../selection/SelectionTargets';
+import { insertTableDataValidation } from './Utils';
 
-const getClipboardRows = (clipboardRows): HTMLElement[] => clipboardRows.get().fold(function () {
-  return;
-}, function (rows) {
-  return Arr.map(rows, function (row) {
-    return row.dom();
-  });
-});
+const getClipboardRows = (clipboardRows): HTMLElement[] => clipboardRows.get().fold(
+  () => {},
```

Comment:
This seems wrong given the return type? Previously this would have returned `undefined`, but that seems incorrect as well and instead, I think this should probably just be `() => [ ]`